Core/src/org/sleuthkit/autopsy/datamodel/AbstractAbstractFileNode.java

```diff
@@ -368,18 +327,16 @@ public String toString() {
             properties.add(new NodeProperty<>(ORIGINAL_NAME.toString(), ORIGINAL_NAME.toString(), NO_DESCR, ""));
         }
 
-        //SCO column prereq info..
-        List<ContentTag> tags = getContentTagsFromDatabase();
-        CorrelationAttributeInstance attribute = getCorrelationAttributeInstance();
-
-        Pair<DataResultViewerTable.Score, String> scoreAndDescription = getScorePropertyAndDescription(tags);
-        properties.add(new NodeProperty<>(SCORE.toString(), SCORE.toString(), scoreAndDescription.getRight(), scoreAndDescription.getLeft()));
-        DataResultViewerTable.HasCommentStatus comment = getCommentProperty(tags, attribute);
-        properties.add(new NodeProperty<>(COMMENT.toString(), COMMENT.toString(), NO_DESCR, comment));
-        if (!UserPreferences.hideCentralRepoCommentsAndOccurrences()) {
-            Pair<Long, String> countAndDescription = getCountPropertyAndDescription(attribute);
-            properties.add(new NodeProperty<>(OCCURRENCES.toString(), OCCURRENCES.toString(), countAndDescription.getRight(), countAndDescription.getLeft()));
-        }
+        // Create place holders for S C O 
+        properties.add(new NodeProperty<>(SCORE.toString(), SCORE.toString(), NO_DESCR, ""));
+        properties.add(new NodeProperty<>(COMMENT.toString(), COMMENT.toString(), NO_DESCR, ""));
+        properties.add(new NodeProperty<>(OCCURRENCES.toString(), OCCURRENCES.toString(), NO_DESCR, ""));
```

Comment:
We should only add the occureence property to sheet set when UserPreferences.hideCentralRepoCommentsAndOccurrences() == false so that the column does not exist when its contents would be always be empty.